frontend/public/components/storage/create-pvc.tsx

```diff
@@ -19,6 +19,8 @@ const NameValueEditorComponent = (props) => (
   />
 );
 
+const cephRbdProvisioner = 'rbd.csi.ceph.com';
```

Comment:
per console naming this would be

```suggestion
const cephRBDProvisioner = 'rbd.csi.ceph.com';
```

Reply:
We might want to add `suffix` so it's clear it's not the full value.

```suggestion
const cephRBDProvisionerSuffix = 'rbd.csi.ceph.com';
```